ChemistryLib/PhreeqcIOData/ReactionRate.h

```diff
@@ -15,13 +15,15 @@
 
 namespace ChemistryLib
 {
+namespace PhreeqcIOData
+{
 struct ReactionRate
 {
     ReactionRate(std::string kinetic_reactant_,
                  std::vector<std::string>
                      expression_statements_)
         : kinetic_reactant(std::move(kinetic_reactant_)),
-          expression_statements(std::move(expression_statements_))
+          expression_statements(expression_statements_)
```

Comment:
why no r-value anymore?

Reply:
As the creation function `createReactionRates` is shared for creating instances of `PhreeqcIOData::ReactionRate` and `PhreeqcKernelData::ReactionRate`.

Reply:
Fixed.